spec/controllers/bikes_controller_spec.rb

```diff
@@ -911,7 +911,7 @@
 
         it 'updates the bike and components' do
           component1 = FactoryBot.create(:component, bike: bike)
-          handlebar_type_id = FactoryBot.create(:handlebar_type).id
+          another_handlebar_type = "bmx"
```

Comment:
would you switch this to be something other than "bmx", so that `another_handlebar_type` is distinct?